doc/cabal-package.rst

```diff
@@ -1345,20 +1340,8 @@ Benchmarks
 The benchmark may be described using the following fields, as well as
 build information fields (see the section on `build information`_).
 
-.. pkg-field:: type: interface (required)
```

Comment:
Is the `type` field no longer recognized, or why was this section deleted?

Reply:
I've added the section back in #8344